content/news/031/index.md

```diff
@@ -74,6 +74,33 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Tooling Updates
 
+### [Blackjack]
+
+![Blackjack: Showcase of the new catmull-clark subdivision](blackjack.gif)
+
+[Blackjack] by @setzer22 is a new procedural modeling application made in Rust,
```

Comment:
Just want to check, but do you want to add any links to your username? Such as Twitter or Github?

Reply:
It's fine as-is :+1: I don't have a twitter with my username and I'm already sharing links to my github projects so people can easily find me there